resources/metadata/npmExecuteScripts.yaml

```diff
@@ -0,0 +1,42 @@
+metadata:
+  name: npmExecuteScripts
+  description: Execute npm run scripts with optional install before
+  longDescription: |
+    Execute npm run scripts in all package json files, if they implement the scripts
+spec:
```

Comment:
I think we have to add an alias to npmExecute to not lose the configuration done there...